airflow/providers/yandex/secrets/secrets_manager.py

```diff
@@ -0,0 +1,280 @@
+
+        self.folder_id = folder_id
+        self.connections_prefix = connections_prefix.rstrip(sep) if connections_prefix is not None else None
+        self.variables_prefix = variables_prefix.rstrip(sep) if variables_prefix is not None else None
+        self.config_prefix = config_prefix.rstrip(sep) if config_prefix is not None else None
+        self.sep = sep
+        self.endpoint = endpoint
+
+    def get_conn_value(self, conn_id: str) -> str | None:
+        """
+        Retrieve from Secrets Backend a string value representing the Connection object.
+
+        :param conn_id: Connection ID
+        :return: Connection Value
+        """
+        if self.connections_prefix is None:
+            return None
+
+        if conn_id == self.yc_connection_id:
+            return None
+
+        return self._get_secret_value(self.connections_prefix, conn_id)
+
+    def get_variable(self, key: str) -> str | None:
+        """
+        Return value for Airflow Variable.
+
+        :param key: Variable Key
+        :return: Variable Value
+        """
+        if self.variables_prefix is None:
+            return None
+
+        return self._get_secret_value(self.variables_prefix, key)
+
+    def get_config(self, key: str) -> str | None:
+        """
+        Return value for Airflow Config Key.
+
+        :param key: Config Key
+        :return: Config Value
+        """
+        if self.config_prefix is None:
+            return None
+
+        return self._get_secret_value(self.config_prefix, key)
+
+    @cached_property
+    def _client(self):
+        """
+        Create a Yandex Cloud SDK client.
+
+        Lazy loading is used here
+        because we can't establish a Connection until all secrets backends have been initialized.
+        """
+        if self.yc_connection_id:
+            self.yc_oauth_token = self._get_field("oauth")
+            self.yc_sa_key_json = self._get_field("service_account_json")
+            self.yc_sa_key_json_path = self._get_field("service_account_json_path")
+            self.folder_id = self.folder_id or self._get_field("folder_id")
+
+        credentials = get_credentials(
+            oauth_token=self.yc_oauth_token,
+            service_account_json=self.yc_sa_key_json,
+            service_account_json_path=self.yc_sa_key_json_path,
+        )
+        sdk_config = self._get_endpoint()
+        return yandexcloud.SDK(user_agent=provider_user_agent(), **credentials, **sdk_config).client
+
+    def _get_endpoint(self) -> dict[str, str]:
+        sdk_config = {}
+
+        if self.endpoint:
+            sdk_config["endpoint"] = self.endpoint
+
+        return sdk_config
+
+    @cached_property
+    def _connection(self) -> Connection | None:
+        if not self.yc_connection_id:
+            return None
+
+        conn = Connection.get_connection_from_secrets(self.yc_connection_id)
+        logging.info("Using connection ID '%s' for task execution.", conn.conn_id)
```

Comment:
Same here with Root logger you either could create new logger in top of the module or inherit class from `airflow.utils.log.logging_mixin.LoggingMixin`



```python
...
from airflow.utils.log.logging_mixin import LoggingMixin
...

class LockboxSecretBackend(BaseSecretsBackend, LoggingMixin):
    ...

    @cached_property
    def _connection(self) -> Connection | None:
        ...
        self.log.info("Using connection ID '%s' for task execution.", conn.conn_id)
```